Formula/docutils.rb

```diff
@@ -17,50 +17,19 @@ class Docutils < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "63d91b072dde1c302435f1c6eea482d3a10cd83d109d2dd12adf192de28ca55f"
   end
 
-  depends_on "python@3.10" => [:build, :test]
-  depends_on "python@3.11" => [:build, :test]
-
-  def pythons
-    deps.map(&:to_formula)
-        .select { |f| f.name.match?(/^python@3\.\d+$/) }
-  end
+  depends_on "python@3.11"
 
   def install
-    pythons.each do |python|
-      python_exe = python.opt_libexec/"bin/python"
-      system python_exe, *Language::Python.setup_install_args(libexec, python_exe)
-
-      site_packages = Language::Python.site_packages(python_exe)
-      pth_contents = "import site; site.addsitedir('#{libexec/site_packages}')\n"
-      (prefix/site_packages/"homebrew-docutils.pth").write pth_contents
-
-      pyversion = Language::Python.major_minor_version(python_exe)
-      Dir.glob("#{libexec}/bin/*.py") do |f|
-        bname = File.basename(f, ".py")
-        bin.install_symlink f => "#{bname}-#{pyversion}"
-        bin.install_symlink f => "#{bname}.py-#{pyversion}"
-      end
+    python3 = "python3.11"
+    system python3, *Language::Python.setup_install_args(prefix, python3)
```

Comment:
I had just simplified by installing into `prefix`, but we could keep in `libexec` if we want to allow users to have run `pip3.11 install docutils`.